### DISCUSSION TITLE
Is there a way to access the current form data in the fields method?

### DISCUSSION THREAD

**Discussion by jaxramus** [laravel/nova-issues#4144] [Q&A] [answered]
How do I get a value that is currently present on the form inside the `fields` method?

Example:

```
Text::make('Company ID', 'company_id'),
```

I've tried obtaining this value by using `$request->company_id` and `$this->company_id`, and neither works.

I need something like the `FormData`, but to use it inside of the fields method rather than dependsOn.

How do I do this?

**Comment by davidhemphill** [comment, marked as answer]
What are you trying to do? Resources delegate calls to the underlying model so you can access its data using `$this->company_id` or `optional($this->resource)->company_id` or similar.
